static/styles/lib/bootstrap/scss/_forms.scss

```diff
@@ -171,6 +171,7 @@ select.form-control-sm {
   padding: $input-padding-y-lg $input-padding-x-lg;
   font-size: $font-size-lg;
   @include border-radius($input-border-radius-lg);
+  hyphens: none;
```

Comment:
Sorry, habe total übersehen, dass die Änderungen hier direkt in Bootstrap gemacht wurden. Dies wollen wir eigentlich vermeiden, damit wir Bootstrap jederzeit einfach updaten können. Verschiebe die Änderung also am besten noch in die `base.scss` ([link](https://github.com/schul-cloud/schulcloud-client/blob/master/static/styles/lib/base.scss))
Also in etwa so:
```scss
/* /static/styles/lib/base.scss */
.form-control .form-control-lg {
  hyphens: none;
}
```